cmd/gateclient/client.go

```diff
@@ -606,3 +595,15 @@ func securePrompt(output func(string), inputMsg string) string {
 	secret := string(byteSecret)
 	return strings.TrimSpace(secret)
 }
+
+func expandFilepath(path string) (string, error) {
```

Comment:
this might make more sense under the util package than under the gate client package?

Reply:
I agree with you. I’ll update it!